src/app/components/AnswersWrapper.js

```diff
@@ -0,0 +1,25 @@
+import { Button } from './Button';
+
+export const AnswersWrapper = (answers, rightAnswer, chosenAnswer) => {
+  const answersWrapperObj = document.createElement('div');
+  answersWrapperObj.classList.add('answersWrapper');
+  answersWrapperObj.id = 'answersWrapper';
+
+  answers.forEach((answer, index) => {
+    const classList = ['button--answer'];
+    const isAnswerCorrect = answer === rightAnswer;
+    isAnswerCorrect
+      ? classList.push('button--correctAnswer')
+      : classList.push('button--incorrectAnswer');
+    const answerButton = Button({
+      id: `answer${index + 1}`,
+      btnText: answer,
+      classList,
+      onClickFn: chosenAnswer(answer, isAnswerCorrect),
+    });
+
+    answersWrapperObj.appendChild(answerButton);
```

Comment:
Odseparowałbym też tworzenie elementu odpowiedzi od jego dodawania, ale to taka kosmetyczna zmiana. Chociaż właśnie funkcja też powinna mieć 1 odpowiedzialność - tutaj ma dwie. Dodaje i tworzy.

```
export const AnswersWrapper = (answers, rightAnswer, chosenAnswer) => {
  const answersWrapperObj = document.createElement('div');
  answersWrapperObj.classList.add('answersWrapper');
  answersWrapperObj.id = 'answersWrapper';

  const answerButtons = answers.map((answer, index) => {
    const isAnswerCorrect = answer === rightAnswer;
    return Button({
      id: `answer${index + 1}`,
      btnText: answer,
      classList: ['button--answer', isAnswerCorrect ? 'button--correctAnswer' : 'button--incorrectAnswer'],
      onClickFn: chosenAnswer(answer, isAnswerCorrect),
    });
  });

  answerButtons.forEach(buttonElement => answersWrapperObj.appendChild(buttonElement))
  return answersWrapperObj;
};
```

Reply:
Done